BCStudentSoftwareDevTeam/celts
# Add upcoming event count to student led accordion.

Added a counter for the upcoming events to the accordion header and removed the extra `Other CELTS` option from the create event page. 

Changed `Dietary Restrictions` text box so that it has a placeholder and if there are no dietary restrictions entered we no longer just say "None".

To Test:
Run the test suite and verify that all the tests pass. 
Run the application and create multiple events in a program, make sure some are in the past and some are canceled, verify the count that is given in the accordion header is accurate to upcoming events that are not canceled.
